src/mono/wasm/timezones.props

```diff
@@ -0,0 +1,12 @@
+<Project>
+  <ItemGroup Condition="'$(TargetOS)' == 'Browser'">
```

Comment:
Instead of conditioning the itemgroup I would condition the import of this file. It's unusual to import an empty msbuild file.